app/src/main/res/values-de/strings.xml

```diff
@@ -185,6 +187,7 @@
     <string name="none">Keine(r)</string>
     <string name="section_issue_labels">Markierungen:</string>
     <string name="log_in">Einloggen</string>
+    <string name="log_out">Ausloggen</string>
```

Comment:
The 'rightest' (omg xD) would be 'Abmelden'. But then we need 'Anmelden' for log_in to..